commands/server.ts

```diff
@@ -147,9 +152,13 @@ export default class ServerCommand extends Command {
   protected startServer(argv: any, project: Project) {
     let bootstrapPath = project.isAddon ? path.join('test/dummy/index.js') : 'index.js';
     let args = [ bootstrapPath ];
-    if (argv.debug) {
+    if (argv.debugBrk) {
       args.unshift('--inspect-brk');
     }
+    if(argv.debug) {
```

Comment:
Would we ever want both of these flags applied?

Reply:
I dont think its a problem. `--inspect --inspect-brk` is equivalent AFAIK. No reason to add a check here.